src/scripts/util.js

```diff
@@ -22,6 +26,14 @@ export const changeImage = (imageElement, imageSrc) => {
   imageElement.setAttribute('src', imageSrc);
 };
 
-export const checkUserInfo = (id, password) => {
-  return id.value === USER_INFO.ID && password.value === USER_INFO.PASSWORD;
+export const checkValueMatch = (firstElement, secondElement) => {
+  return firstElement.value === secondElement.value;
+};
+
+export const checkUserInfo = (idElement, passwordElement) => {
+  return idElement.value === USER_INFO.ID && passwordElement.value === USER_INFO.PASSWORD;
+};
```

Comment:
idElement, passwordElement를 인자로 받지 않고 아예 string으로 id ,password만 받도록 개선해보면 어떨까요?

요 함수에서는 id나 password가 element라는걸 굳이 알필요 없을 것 같아요!

```suggestion
export const checkUserInfo = (id, password) => {
  return id=== USER_INFO.ID && password === USER_INFO.PASSWORD;
};
```

Reply:
34326528aa0809990f2d67307361bc4017951fe3 정정했습니다!
checkUserId 함수도 동일하게 element 를 사용할 필요가 없을 거 같아 함께 정정했습니다~!